packages/@coorpacademy-components/src/molecule/skill-picker-modal/test/fixtures/default.js

```diff
@@ -0,0 +1,43 @@
+export default {
+  props: {
+    skills: [
+      {
+        skillTitle: 'Creativity and innovation',
+        focus: false
```

Comment:
The skillRef should be kept so we can inject it in the onConfirm handler

(Note: focus can be them view as a list of active refs)